app/src/question/index.js

```diff
@@ -53,20 +50,26 @@ module.exports = rust.class({
 
     var counts = Array(q.choices.length);
     var late = Array(q.choices.length);
+    var overallVotes = Array(q.choices.length);
     counts.fill(0);
     late.fill(0);
+    overallVotes.fill(0);
     var total = 0;
     var lateTotal = 0;
     _.each(this.props.votes, function(v) {
       if (timeUtil.timeBetween(q.created, v.created) < voteDuration) {
         counts[v.choiceIndex] += 1;
+        overallVotes[v.choiceIndex] += 1;
```

Comment:
This line is identical to line 66. Generally copy+paste / duplications means something could be improved. Generally. Like in this case. Not the biggest deal but remember that. I'll approve anyway.